### DISCUSSION TITLE
Does `iconRotate` of `SymbolLayer` support Transition?

### DISCUSSION THREAD

**Discussion by abdelhameedhamdy** [maplibre/maplibre-react-native#573] [Q&A] [answered]
Hey, 

Is it applicable to have an animation (transition) on **iconRotate** prop in **SymbolLayer** style ? if so, any thoughts how to achieve it?

Thanks

**Comment by KiwiKilian** [comment, marked as answer]
`icon-rotate` is not marked as transitionable on the [MapLibre Style Spec.](https://maplibre.org/maplibre-style-spec/layers/#icon-rotate) Therefore you would have to discuss this feature first there, then it needs to be implemented on MapLibre Native, before this wrapper could support it.
